rust/README.md

```diff
@@ -1,13 +1,31 @@
 # Echo Bot - Rust
 
-> Warning: the rust API is **unstable** and it is subject to change, use the **cffi api** if you want more stability and be careful when updating the core as the api might have been changed.
+> Warning: the rust API is **unstable** and it is subject to change,
+> use the **cffi api** in the `deltachat_ffi` crate if you want more
```

Comment:
IIRC @hpk42 and others were talking about not encouraging the CFFI API for Rust development